build/bin/test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -xe
+
+echo "TODO"
```

Comment:
Can we add actual test call here?
```suggestion
npm run test && npm run test-spec
```

Reply:
Well, this would require downloading all submodules first:
```
git submodule update --init --recursive
```
I guess we can keep as is - we're not going to iterate on that any time soon